cmd/ocm/create/cluster/cmd.go

```diff
@@ -39,11 +39,16 @@ import (
 	"github.com/spf13/pflag"
 )
 
+var ValidSubscriptionTypes = []string{standardSubscriptionType, marketplaceRhmSubscriptionType, marketplaceGcpSubscriptionType}
+
 const (
 	defaultIngressRouteSelectorFlag            = "default-ingress-route-selector"
 	defaultIngressExcludedNamespacesFlag       = "default-ingress-excluded-namespaces"
 	defaultIngressWildcardPolicyFlag           = "default-ingress-wildcard-policy"
 	defaultIngressNamespaceOwnershipPolicyFlag = "default-ingress-namespace-ownership-policy"
+	standardSubscriptionType                   = "standard"
+	marketplaceRhmSubscriptionType             = "marketplace-rhm"
+	marketplaceGcpSubscriptionType             = "marketplace-gcp"
```

Comment:
This will map to existing billing models. 

The possible values are listed here: https://github.com/openshift-online/ocm-api-model/blob/main/model/accounts_mgmt/v1/billing_model_type.model#L18-L40